spark-test/src/test/resources/test-catalog/attributes/all-ones___8___metadata.json

```diff
@@ -0,0 +1 @@
+[{"name":"all-ones","zoom":8},{"keyBounds":{"minKey":{"col":1,"row":1},"maxKey":{"col":6,"row":7}},"schema":null,"metadata":{"cellType":"float32","extent":{"xmin":-135.00000125,"ymin":-89.99999,"xmax":134.99999125,"ymax":67.49999249999999},"layoutDefinition":{"extent":{"xmin":-180.0,"ymin":-89.99999,"xmax":179.99999,"ymax":89.99999},"tileLayout":{"layoutCols":8,"layoutRows":8,"tileCols":3,"tileRows":4}},"crs":"+proj=longlat +datum=WGS84 +no_defs "},"header":{"keyClass":"geotrellis.spark.SpatialKey","valueClass":"geotrellis.raster.Tile","path":"file:/home/likewise-open/AZVA-INT/jmcclain/local/src/geotrellis/spark/src/test/resources/test-catalog/all-ones/8"},"keyIndex":{"obj":"rO0ABXNyADpnZW90cmVsbGlzLnNwYXJrLmlvLmluZGV4LnJvd21ham9yLlJvd01ham9yU3BhdGlhbEtleUluZGV4DTH0g5hxQLACAANJAApsYXlvdXRDb2xzSQAGbWluQ29sSQAGbWluUm93eHAAAAAGAAAAAQAAAAE="}}]
```

Comment:
some gitignore got messed up. We shouldn't be committing test catalog. In fact we shouldn't even need to be generating this file. I think `catalog-test` will be written from the actual hadoop catalog spec where as this file used to be the source of layers for the tests. Layers for the test are now generated and kept in memory.